lib/sidekiq-scheduler/manager.rb

```diff
@@ -18,21 +18,28 @@ def initialize(options)
       Sidekiq::Scheduler.enabled = options[:enabled]
       Sidekiq::Scheduler.dynamic = options[:dynamic]
       Sidekiq::Scheduler.listened_queues_only = options[:listened_queues_only]
-      Sidekiq.schedule = options[:schedule]
+
+      @schedule = options[:schedule]
+      @enabled = options[:enabled]
     end
 
     def stop
       Sidekiq::Scheduler.clear_schedule!
     end
 
     def start
-      Sidekiq::Scheduler.load_schedule!
+      if enabled
+        Sidekiq.schedule = schedule
+        Sidekiq::Scheduler.load_schedule!
```

Comment:
You're right, that flag is needed by `reload_schedule!`

How about conditionally invoking `Sidekiq.schedule = ...` if enabled? Doing that way `#load_schedule!` we'll still have an extra conditional but will not need for extra instance variables, and it will also log `'SidekiqScheduler is disabled'` message.

```ruby
    def initialize(options)
      # ...
      Sidekiq.schedule = options[:schedule] if Sidekiq::Scheduler.enabled
    end
```
Just an idea :)